doc/openapi.yaml

```diff
@@ -0,0 +1,117 @@
+openapi: 3.0.1
```

Comment:
Swagger documentation should be validated with @dietertaotesting to make sure they get parsed corectly